### DISCUSSION TITLE
Error in the SelectItem.vue and SelectContent.vue subcomponents?

### DISCUSSION THREAD

**Discussion by Cristian024** [unovue/shadcn-vue#632] [Q&A]
Today, I downloaded the "Select" component but when I tried to use it I got an error, checking the subcomponents I found these errors:

![imagen_2024-06-23_195228995](https://github.com/radix-vue/shadcn-vue/assets/110924565/a0c209b8-ddea-4e51-adc7-0d82b81cbe48)

![image](https://github.com/radix-vue/shadcn-vue/assets/110924565/40d43f64-1f0c-4061-819d-c3ccef2046cc)

Is there a bug or did I install the component wrong? :p

**Comment by zernonia** [comment]
Upstream issue. Related ticket https://github.com/Dunqing/detype/pull/3
